source/js/components/pulse-project-list/pulse-project-list.scss

```diff
@@ -20,10 +20,13 @@
 
   .project-image {
     width: 100%;
-    max-height: none;
```

Comment:
This was here to override the [now-obsolete rule](https://github.com/mozilla/foundation.mozilla.org/pull/6514/files#diff-9831057300166c3b2df655ac821251dde9350c4cc319679818cd76bf711e5244L1-L3) 
```scss
.project-image {
  max-height: 11em;
}
```